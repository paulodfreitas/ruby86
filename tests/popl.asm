.module m0
.pseg
;Teste da semantica da instrução Popl para o Y86
	irmovl $100,%esp  
	irmovl $ABCD,%eax 
	pushl  %eax         
	popl   %esp         
	halt
.end
