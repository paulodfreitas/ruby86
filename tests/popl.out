@ 0000
30
84
00
01
00
00
30
80
cd
ab
00
00
a0
08
b0
48
10
@ 0011
